src/improvements/tasks/sep/123-opcm-upgrade-superchainconfig-v410-freya-u15/addresses.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can you change these tasks to be `024`, `025` and `026` unless there's a reason you're adding 100 to these prefixes.

Reply:
Sure yeah, I saw the name convention is adding bn now in the name, with eris we used the 1XX and then closed the PR without merging.